README.md

```diff
@@ -83,7 +85,9 @@ Company                                           | Business/products
 [Toggl](https://www.toggl.com/)                   | Time Tracking Tool                       | Go, Ruby, PostgreSQL, Mobile, Web              | ✗                                    | Anywhere                | :keyboard: [Jobs](https://jobs.toggl.com/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Toggl-EI_IE734060.11,16.htm)
 [Toptal](https://www.toptal.com/)                 | Tech freelancer marketplace              | Ruby, Web                                      | ✗                                    | Anywhere                | :muscle: [Core team positions](https://www.toptal.com/careers)<br>:computer: [Apply as a developer](https://www.toptal.com/developers)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Toptal-EI_IE882070.11,17.htm)
 [Trello](https://trello.com)                      | Trello                                   | Node, Mobile, Web                              | ✓                                    | Anywhere                | :keyboard: [Jobs](https://trello.com/jobs)<br>:door: [Glassdoor (acquired by Atlassian)](https://www.glassdoor.com.au/Overview/Working-at-Atlassian-EI_IE115699.11,20.htm)
-[Tyk](https://tyk.io/)                            | Cloud native API Management, open source | Go, Web, DevOps                                | ✓                                    | UK, US, Singapore       | :keyboard: [Jobs](https://tyk.io/about/team/)<br>:door: [Glassdoor ](https://www.glassdoor.co.uk/Reviews/Tyk-Reviews-E2321465.htm)
+[Twitter](htt[s://twitter.com])                   | Social media platform                    | C++, Go, Java, JavaScript, Python, Scala etc   | 
```

Comment:
Twitter seems more committed, but can you please fix the link and ensure that this renders correctly?